sdk/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpansProcessor.java

```diff
@@ -411,4 +324,277 @@ public void run() {
       }
     }
   }
+
+  @Immutable
+  @AutoValue
+  public abstract static class Config {
+
+    public abstract boolean isReportOnlySampled();
+
+    public abstract long getScheduleDelayMillis();
+
+    public abstract int getMaxQueueSize();
+
+    public abstract int getMaxExportBatchSize();
+
+    public abstract int getExporterTimeoutMillis();
+
+    public static Config getDefault() {
+      return new Builder().build();
+    }
+
+    /**
+     * Creates a {@link Config} object reading the configuration values first from the environment
+     * and then from system properties. If a configuration value is missing, it uses the default
+     * value.
```

Comment:
Can we add a wee bit of clarification that the system property will override the environment values?